### DISCUSSION TITLE
Mito variants

### DISCUSSION THREAD

**Discussion by AlessandroLas** [broadinstitute/seqr#4346] [Q&A]
Hi all,

I'd like to analyze mitochondrial variants on local install of seqr.
However, I couldn't find any docs.
Is it necessary to download mito reference data?
How to annotate mito variants?
What does it mean Mitochondria Caller in the "Edit dataset"?

Thanks.
Alessandro

**Comment by hanars** [comment]
Hi Alessandro, Unfortunately this functionality is not available for local installs yet. It will be made available sometime in 2025

**Comment by AlessandroLas** [comment]
Thank you Hana.
Wiil wait for updates.
